cardano-cli/src/Cardano/CLI/Read.hs

```diff
@@ -721,6 +721,8 @@ readWitnessSigningData (KeyWitnessSigningData skFile mbByronAddr) = do
           -- A Byron address should only be specified along with a Byron signing key.
           Left ReadWitnessSigningDataSigningKeyAndAddressMismatch
   where
+    -- If you update these variables, consider updating the ones with the same
+    -- names in Cardano.CLI.Read
```

Comment:
This comment is confusing - it's `Cardano.CLI.Read` module.

Reply:
Yep, copy/paste error. good catch! Corrected :+1: